### ISSUE TITLE
Sprues don't allow negative coordinates

### ISSUE THREAD

**Issue by rnewton** [andymeneely/squib#336]
I'm trying to setup a sprue for [PrintPlayGames](https://www.printplaygames.com/prototypes/formatting-guidelines/card-formatting-templates/) card templates, but I'm not able to get a y-value that lines up with their template. You'll note that they have the cards rotated. 

Using this as the sprue:

```yml
---
sheet_width: 12in
sheet_height: 18in
card_width: 2.5in
card_height: 3.5in
cards:
- x: 8.375in
  y: 0in
```

Results in 

<img width="722" alt="Screen Shot 2021-08-01 at 11 18 43 AM" src="https://user-images.githubusercontent.com/445976/127781301-386d913c-4acb-4f38-9073-19147c6fe8bb.png">

If I rotate the card, it shifts down, indicating that the `position_reference` isn't lining up. Sprue: 

```yml
---
sheet_width: 12in
sheet_height: 18in
card_width: 2.5in
card_height: 3.5in
cards:
- x: 8.375in
  y: 0in
  rotate: :clockwise
```

Results in 

<img width="721" alt="Screen Shot 2021-08-01 at 11 18 06 AM" src="https://user-images.githubusercontent.com/445976/127781289-6f5f8097-10bc-48f5-8c0c-6e4dd6c9987a.png">

As a workaround, I can probably set the `position_reference` to `:center`, but that's a little harder to reason about card placement. Would be nice if the rotation wasn't offset somehow for the `:topleft` or if I was allowed to set a negative position for `x/y`. If I do, I don't match the regex:

```
Squib::Sprues::InvalidSprueDefinition: Invalid sprue definition in file: src/layout/ppg-poker.yml. "cards"[0]["y"] is not a String matching /^(\d*[.])?\d+(in|cm|mm)$/
```

**Comment by andymeneely** [comment]
Fixed! Upgrade to v0.17.1 and you'll be able to get negative coordinates.

**Comment by rnewton** [comment]
Thanks, Andy!
